lib/model/smartdeviceplugin.py

```diff
@@ -185,6 +186,12 @@ def __init__(self, sh, logger=None, **kwargs):
 
         self._shtime = Shtime.get_instance()
 
+        #resend
+        self._parameters[PLUGIN_ATTR_SEND_RETRIES] = self.get_parameter_value(PLUGIN_ATTR_SEND_RETRIES)
+        self._parameters[PLUGIN_ATTR_SEND_RETRIES_CYCLE] = self.get_parameter_value(PLUGIN_ATTR_SEND_RETRIES_CYCLE)
+        # Set protocol to resend if send_retries is > 0
+        if self._parameters.get(PLUGIN_ATTR_SEND_RETRIES, 0) > 0:
```

Comment:
ergänze `and not self._parameters.get[PLUGIN_ATTR_PROTOCOL]:`

Wenn schon ein protocol gesetzt ist, sollte das nicht automatisch überschrieben werden. Vielleicht implementiert es ja resend selbst? Manuell geht immer vor Automatik.